build_releases

```diff
@@ -21,14 +21,17 @@ fi
 
 eval $(go env)
 
-export GOBIN=${PWD}/bin/releases
+if [ -z ${BIN_PATH+a} ]; then
+	export BIN_PATH=${PWD}/bin/${GOARCH}
```

Comment:
Previously this was output in `${PWD}/bin/releases`

Reply:
good catch